### PR TITLE
Add i18n  - Flowbroker

### PR DESCRIPTION
**Pull request by mprevide**
* **Please check if the PR fulfills these requirements**
- [ ] Docs have been added / updated (for bug fixes / features)

- [ ] Tests for the changes have been added (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
Add i18n  in  Flowbroker

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
* This PR is connected to [dojot/dojot#898](https://github.com/dojot/dojot/issues/898).

* **Other information**:

### CODE REVIEW

**Comment by raulnegreiros** [comment]
This PR is mixed with dojot/flowbroker#118

**Comment by mprevide** [comment]
It will need some changes

File: orchestrator/nodes/switch/locales/pt-BR.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      "nooverride": "Aviso: as propriedades msg não podem mais substituir as propriedades do nó definido. Consulte bit.ly/nr-override-msg-props"
+    }
+  },
+  "switch": {
+    "label": {
+      "property": "Propriedade",
+      "rule": "regra"
+    },
+    "and": "e",
+    "checkall": "verificando todas as regras",
```
Comment (raulnegreiros) [inline orchestrator/nodes/switch/locales/pt-BR.json]: "verificar todas as regras"

File: orchestrator/nodes/switch/locales/pt-BR.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    "stopfirst": "parando após o primeiro",
```
Comment (raulnegreiros) [inline orchestrator/nodes/switch/locales/pt-BR.json]: "parar após a primeira correspondência"

File: orchestrator/nodes/template/locales/pt-BR.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "template": "modelo",
+  "label": {
+    "name": "Nome",
+    "template": "Modelo",
+    "property": "Propriedade",
+    "syntax": "Formato",
+    "output": "Saída",
+    "handlebars": "Handlebars modelo",
+    "plain": "Texto simples",
+    "json": "Parsed JSON"
```
Comment (raulnegreiros) [inline orchestrator/nodes/template/locales/pt-BR.json]: "JSON analisado"

File: orchestrator/nodes/remoteNode/index.js [hunk 4]
```diff
@@ -18,6 +18,7 @@ class RemoteNodeHandler extends dojot.DataHandlerBase {
     this.serverAddress = serverAddress;
     this.serverPort = serverPort;
     this.dispatcher = null;
+    this.getLocaleData = this.getLocaleData.bind(this);
```
[the remark this answers is not in the document]
Comment (mprevide) [reply orchestrator/nodes/remoteNode/index.js]: For nothing hahaha

**Pull request merged**